packages/flutter_tools/test/src/common.dart

```diff
@@ -145,6 +145,22 @@ Future<void> expectToolExitLater(Future<dynamic> future, Matcher messageMatcher)
   }
 }
 
+Matcher containsIgnoreWhitespace(String toSearch){
```

Comment:
nit. In terms of English grammar, I think we should call this `containsIgnoringWhitespace`.

Reply:
done!